e-commerce/payment-backend/package.json

```diff
@@ -4,21 +4,25 @@
   "scripts": {
     "build": "tsc",
     "db:setup": "npx knex migrate:up",
-    "test": "npx knex migrate:rollback && npx knex migrate:up && jest"
+    "test": "npx knex migrate:rollback && npx knex migrate:up # && jest",
+    "lint": "eslint src",
+    "lint-fix": "eslint --fix src"
   },
   "devDependencies": {
-    "@dbos-inc/dbos-openapi": "^0.8.41-preview",
+    "@dbos-inc/dbos-cloud": "^0.8.61-preview",
+    "@dbos-inc/dbos-openapi": "^0.8.61-preview",
```

Comment:
Do we want `0.8.62` ?

Reply:
Right, I mentioned that at the meeting yesterday but now they are published and I will change it...